.github/workflows/release.yml

```diff
@@ -40,8 +40,16 @@ jobs:
         run: gh release view ${{github.ref_name}} --json body --jq '.body' > releaseBody.txt
 
       - name: Add Jira links
+        shell: python
         run: |
-          sed -i -e 's/\[\(RHEL.*\)\]/[[\1](https:\/\/issues.redhat.com\/browse\/\1)]/' releaseBody.txt
+          import re
+
+          regex=re.compile(r"(?<!\/)(RHELC?-\d+)\b(?!\]\()", re.MULTILINE)
+          with open("releaseBody.txt", 'r+') as f:
+            data = f.read()
+            f.seek(0)
+            f.write(regex.sub(r"[\g<0>](https://issues.redhat.com/browse/\g<0>)", data))
```

Comment:
This is always a nightmare to understand... Can we move this regex to its own line and only pass the result to the write function?

Reply:
Sure! Do you want us to annotate it too? It's essentially just `\1 url \1` to grab the first group

Reply:
Yeah, I think that would be a nice idea. Even though it seems very simple, having an annotation of what is the outcome might be good as well.

Reply:
Fixed!